README.md

```diff
@@ -15,7 +15,7 @@ function getBlockByHash(hash: string): Promise<Block|null> {
         body: { jsonrpc: "2.0", id: 1, method: "eth_getBlockByHash", params: [hash, false] }
     }).then(response => response.json());
 }
-//function getBlockByHashCallbackStyle(hash: string, callback: (block: Block|null) => void): void {
+//function getBlockByHashCallbackStyle(hash: string, callback: (error: Error, block?: Block) => void): void {
```

Comment:
I lied, that isn't right either.  I fixed both on master:
`error?: Error, block?: Block|null`
`error?: Error, logs?: Log[]`

A `null` `Block` is a valid non-error result, but if there are no logs then the result should be an empty `Log[]`.  In both cases either Error _or_ block/log must be defined and the other must be undefined (hence why they are both marked as optional).